README.md

```diff
@@ -35,6 +35,58 @@ The software consists of three main components:
 * Precise Orbit Determination (POD), and
 * Various scripts for combination and analysis of solutions
 
+
+## Getting started
+
+You can quickly download a ready-to-run Ginan environment using docker by running:
+
+    mkdir ginan-data
+    docker run -it -v ginan-data:/data gnssanalysis/ginan:v3.0.0 bash
+
+This command connects the `ginan-data` directory on the host (your pc), with the `/data` directory in the container, to allow file access between the two systems, and opens a command line (`bash`) for executing commands.
+
+You will need to have [docker](https://docs.docker.com/get-docker/) installed to use this method.
+
+To verify you have the Ginan executables available once at the Ginan command line, run:
+
+    pea --help
+
+
+### Download example dependencies
+
+Download all of the example data using the scripts and filelists provided. From `/ginan`:
+
+```bash
+cd inputData/data
+./getData.sh
```

Comment:
These scripts will download into the `/ginan` directory, which is not mapped to the local machine (and therefore will disappear once the terminal is closed). 

This may be ok for the inputs if that's what we want, but the output directory also points at locations within `/ginan`. 
Just a note to the user in the README would be fine at this point, but we may want to change this a bit so that outputs go to the `ginan-data` directory that you've created.